infra/gcb/builds_status.py

```diff
@@ -66,16 +89,40 @@ def upload_status(successes, failures, status_filename):
 
 
 def is_build_successful(build):
+  """Check build success."""
   return build['status'] == 'SUCCESS'
 
 
+def upload_log(build_id):
+  """Uploads log file oss-fuzz-build-logs."""
+  status_bucket = _get_storage_client().get_bucket(STATUS_BUCKET)
+  gcb_bucket = _get_storage_client().get_bucket(build_project.GCB_LOGS_BUCKET)
+  log_name = 'log-{0}.txt'.format(build_id)
+  log = gcb_bucket.blob(log_name)
+  dest_log = status_bucket.blob(log_name)
+
+  if not log.exists():
+    print('Failed to find build log {0}'.format(log_name), file=sys.stderr)
+    return False
+
+  if dest_log.exists():
+    return True
+
+  with tempfile.NamedTemporaryFile() as file:
+    log.download_to_filename(file.name)
```

Comment:
Let's fix this to use bucket.copy_blob so we don't have have to download the file. See https://googleapis.dev/python/storage/latest/buckets.html